pennylane_lightning/core/src/simulators/lightning_gpu/gates/cuGates_host.hpp

```diff
@@ -0,0 +1,1426 @@
+// Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
Why did we lose the following?
```suggestion
// Copyright 2022-2023 Xanadu Quantum Technologies Inc.

// Licensed under the Apache License, Version 2.0 (the "License");
```

Reply:
Nice catch, done!

Reply:
We should also make sure to check all license headers and preserve the ones that NVIDIA added with the changes that were included from their PRs.